starboard/testing/fake_graphics_context_provider.cc

```diff
@@ -174,7 +185,9 @@ void FakeGraphicsContextProvider::InitializeWindow() {
 }
 
 void FakeGraphicsContextProvider::InitializeEGL() {
-  display_ = EGL_CALL_SIMPLE(eglGetDisplay(EGL_DEFAULT_DISPLAY));
+  display_ = EGL_CALL_SIMPLE(eglGetPlatformDisplay(
+      EGL_PLATFORM_ANGLE_ANGLE, reinterpret_cast<void*>(SB_EGL_DEFAULT_DISPLAY),
```

Comment:
Just a thought that this call (eglGetPlatformDisplay() with this EGL_PLATFORM_ANGLE_ANGLE parameter) is supported as an extension (https://source.chromium.org/chromium/chromium/src/+/main:third_party/angle/extensions/EGL_ANGLE_platform_angle.txt;l=63-65 which may or may not be supported by the platform. 

This code (https://source.chromium.org/chromium/chromium/src/+/main:third_party/angle/src/libANGLE/validationEGL.cpp;l=783-826;drc=883f6246c6bdd5751eb78f48b6c365158b23c3a2) might help figuring out other combinations?